includes/html/pages/device/ports/xdsl.inc.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use App\Models\Port;
+
+echo "<div style='margin: 5px;'><table border=0 cellspacing=0 cellpadding=5 width=100%>";
+
+echo '<tr><th>Port</th><th>Traffic</th><th>Sync Speed</th><th>Attainable Speed</th><th>Attenuation</th><th>SNR Margin</th><th>Output Powers</th></tr>';
+$i = '0';
+
+$ports = Port::join('ports_adsl', 'ports.port_id', 'ports_adsl.port_id')
+    ->where('ports.deleted', '0')
+    ->orderby('ports.ifIndex', 'ASC')
+    ->get();
+
+foreach ($ports as $port) {
+    include 'includes/html/print-interface-adsl.inc.php';
+    $i++;
+}
+
+$ports = Port::join('ports_vdsl', 'ports.port_id', '=', 'ports_vdsl.port_id')
```

Comment:
I suggest joining both tables to reduce queries (or not needed if you merge ports_adsl and ports_vdsl)